build-system/npm-publish/utils.js

```diff
@@ -13,6 +13,27 @@ function getExtensions() {
   return extensions;
 }
 
+/**
+ * Get bento components to be published on npm
+ * @return {Array<any>}
+ */
+function getComponents() {
+  const bundles = require('../compile/bundles.config.bento.json');
+  const components = bundles.map((bundle) => ({
+    'extension': bundle.name,
+    'version': bundle.version,
+  }));
+  return components;
+}
+
+/**
+ * Get bento components and extensions to be published on npm
+ * @return {Array<any>}
+ */
+function getExtensionsAndComponents() {
```

Comment:
fmi: in this context, whats the difference between a "component" and an "extension"?